jenkins/envs/daint_clang.sh

```diff
@@ -5,8 +5,9 @@ source $(dirname "$BASH_SOURCE")/daint.sh
 module load /users/vogtha/modules/compilers/clang/7.0.1
 module load gcc
 
+export GTCMAKE_GT_PREFER_CLANG_CUDA_OVER_NVCC_CUDA=OFF
```

Comment:
why?

Reply:
The clang version which is used here (in my home directory) doesn't work with clang cuda. Also previously this plan was for testing nvcc with clang as host compiler, so now it tests the same thing as before.